packages/dd-trace/src/instrumenter.js

```diff
@@ -112,33 +129,38 @@ class Instrumenter {
           value: true,
           configurable: true
         })
-      })
-    })
 
-    shimmer.massWrap.call(this, nodules, names, function (original, name) {
-      const wrapped = wrapper(original, name)
-      const props = Object.getOwnPropertyDescriptors(original)
-      const keys = Reflect.ownKeys(props)
+        if (Reflect.ownKeys(nodule[name]).includes('_datadog_wrapped')) {
+          nodule[name]._datadog_wrapped = wrapper(nodule[name]._datadog_wrapped)
```

Comment:
This doesn't copy the properties/symbols as is done below.

Reply:
The properties/symbols are already copied onto `nodule[name]` in the prepatching step (in fact, it does this using the code below).